sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,11 +1,18 @@
 # Release History
 
+## 7.14.0-beta.1 (Unreleased)
+
+### Bugs Fixed
+
+- Fixed issue where receiving messages from `ServiceBusSessionReceiverAsyncClient` would never complete.
```

Comment:
Thank you Connie! I created a [Git Issue](https://github.com/Azure/azure-sdk-for-java/issues/34597) describing the root cause analysis you did. Do you mind linking it to the Changelog (as we usually do for reliability fixes). thanks!